testsuite/integration-arquillian/servers/auth-server/jboss/common/jboss-cli/keycloak-server-subsystem.cli

```diff
@@ -21,3 +21,6 @@ echo ** Adding provider **
             module:org.keycloak.testsuite.integration-arquillian-testsuite-providers \
     ] \
 )
+
+echo ** Adding max-detail-length to eventsStore spi **
```

Comment:
I would avoid setting the default in standalone.xml - especially since it's actually different to the default in the code

Reply:
This one is in testsuite and only serves for testing purposes.